infrastructure-playbooks/purge-cluster.yml

```diff
@@ -423,18 +423,6 @@
 
   tasks:
 
-  - name: stop ceph.target with systemd
-    service:
-      name: ceph.target
-      state: stopped
-      enabled: no
-    when: ansible_service_mgr == 'systemd'
-
-  - name: check for anything running ceph
```

Comment:
Why? I still think it's a valuable task, we've seen many issues when systemd resources failed to stop daemons.